### DISCUSSION TITLE
Admin UI amount of clients does not match

### DISCUSSION THREAD

**Discussion by timurkh-devstark** [socketio/socket.io#4476] [Q&A] [answered]
Hi! I started Performance Load test and as you can see on this screen, clients of websocket and amount of sockets in "namespaces" block does not match.
![image](https://user-images.githubusercontent.com/98535865/191739494-4e8b0d5f-46f5-4976-ac72-59e985f7a984.png)

Can you please explain how "Clients" block count clients and what the difference between clients and sockets section in admin ui?

**Comment by darrachequesne** [comment, marked as answer]
Hi! This can happen if the clients are connected at the Engine.IO level (low-level), but not at the Socket.IO level.

With curl:

```
$ curl "http://localhost:3000/socket.io/?EIO=4&transport=polling"
< 0{"sid":"9yeDRtebUcmsY6lvv0T2","upgrades":["websocket"],"pingInterval":300,"pingTimeout":200,"maxPayload":1000000}

// the Engine.IO connection is now considered open

$ curl -X POST "http://localhost:3000/socket.io/?EIO=4&transport=polling&sid=9yeDRtebUcmsY6lvv0T2" -d "40"
< ok
 
// the Socket.IO connection is now considered open (you have entered the main namespace)
```

In your case, it seems the clients cannot reach the main namespace, is there a middleware preventing the connection?

Reference: https://socket.io/docs/v4/middlewares/

**Reply by timurkh-devstark** [reply]
What about clients on http long polling transport protocol? Its always 0, but if I open Clients page I can see clients with http long polling protocol
